pylutron/__init__.py

```diff
@@ -298,7 +301,8 @@ def _parse_keypad(self, keypad_xml, device_group):
                     name=keypad_xml.get('Name'),
                     keypad_type=keypad_xml.get('DeviceType'),
                     location=device_group.get('Name'),
-                    integration_id=int(keypad_xml.get('IntegrationID')))
+                    integration_id=int(keypad_xml.get('IntegrationID')),
+                    uuid=int(keypad_xml.get('UUID')))
```

Comment:
why not store them as strings? You don't need it to be numeric as far as i can tell.

Reply:
Fair point.  Changed to just store the string UUID.  Also added a comment for the GUID.